### DISCUSSION TITLE
Show feature name in QField line layer

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5102] [Q&A]
Hello,
I'm encountering an issue in QField with a line layer in my project. After adding a new line, when I select the created line, instead of seeing the assigned name, I'm presented with the number of records in my database. Interestingly, the point layer in the same project displays the assigned name correctly, with identical attribute configurations in properties.

Does anyone have any insights on how to resolve this problem?

Thank you.

**Comment by suricactus** [comment]
Please get familiar with feature display settings: https://docs.qgis.org/3.4/en/docs/user_manual/working_with_vector/vector_properties.html#display-properties
